CONTRIBUTING.rst

```diff
@@ -202,6 +202,7 @@ their individual contributions.
 * `Cory Benfield <https://www.github.com/Lukasa>`_
 * `Cristi Cobzarenco <https://github.com/cristicbz>`_ (cristi@reinfer.io)
 * `Damon Francisco <https://github.com/dtfrancisco>`_ (damontfrancisco@yahoo.com)
+* `Daniel J. West <https://github.com/danieljwest>`_ 
```

Comment:
Line has trailing whitespace.

Reply:
Thanks! I've added the tests, support for the size 1 case, and (hopefully) removed the whitespace. I'll watch the tests and fix whatever fails